### ISSUE TITLE
Unhandled exception (Application.Current.DispatcherUnhandledException)

### ISSUE THREAD

**Issue by CrazyFish444** [RatScanner/RatScanner#168]
**Error**
Unhandled exception (Application.Current.DispatcherUnhandledException)
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at RatScanner.PageSwitcher.OnClosed(EventArgs e)
   at System.Windows.Window.InternalClose(Boolean shutdown, Boolean ignoreCancel)
   at System.Windows.Application.DoShutdown()
   at System.Windows.Application.ShutdownImpl()
   at System.Windows.Application.ShutdownCallback(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```
<details>
<summary>Log</summary>

```
[22:40:59.5430382] > [Warning] Config version (-1) is not supported!
[22:40:59.5593381] > [Info]  Detected 2560x1440 Resolution.

You can adjust this inside the settings.
[22:40:59.5756556] > [Error] Unhandled exception (Application.Current.DispatcherUnhandledException)
 -------------------- 
 System.NullReferenceException: Object reference not set to an instance of an object.
   at RatScanner.PageSwitcher.OnClosed(EventArgs e)
   at System.Windows.Window.InternalClose(Boolean shutdown, Boolean ignoreCancel)
   at System.Windows.Application.DoShutdown()
   at System.Windows.Application.ShutdownImpl()
   at System.Windows.Application.ShutdownCallback(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
[22:41:03.4251962] > [Warning] Config version (-1) is not supported!
[22:41:03.4419497] > [Info]  Detected 2560x1440 Resolution.

You can adjust this inside the settings.
[22:41:03.4561810] > [Error] Unhandled exception (Application.Current.DispatcherUnhandledException)
 -------------------- 
 System.NullReferenceException: Object reference not set to an instance of an object.
   at RatScanner.PageSwitcher.OnClosed(EventArgs e)
   at System.Windows.Window.InternalClose(Boolean shutdown, Boolean ignoreCancel)
   at System.Windows.Application.DoShutdown()
   at System.Windows.Application.ShutdownImpl()
   at System.Windows.Application.ShutdownCallback(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

```
</details>

**Comment by CrazyFish444** [comment]
I have the webview downloaded and the other 2 files that were listed in the FAQ aswell and this issue is still persisting any help is appreciated!

**Comment by Blightbuster** [comment]
Hey, it seems like RatScanner is having trouble reading / writing files.
- Did you unzip the downloaded archive?
- Is there any antivirus blocking RatScanner?
- Try starting RatScanner as administrator

**Comment by CrazyFish444** [comment]
> Hey, it seems like RatScanner is having trouble reading / writing files.
> 
> * Did you unzip the downloaded archive?
> * Is there any antivirus blocking RatScanner?
> * Try starting RatScanner as administrator

I unzipped using both winrar and 7zip ill try running as an administrator and then update then!

**Comment by CrazyFish444** [comment]
> Hey, it seems like RatScanner is having trouble reading / writing files.
> 
> * Did you unzip the downloaded archive?
> * Is there any antivirus blocking RatScanner?
> * Try starting RatScanner as administrator

No luck on running as admin, I dont have any antivirus other then the one that comes with windows and im almost certain that its not interfering with it. Any other ideas? When I first got it I was able to open it but I wasnt in windowed and it gave a different error when whenever I went to search an item. I closed both then it started doing this so I dont know what exactly happened between then and now. Thanks for taking time to help!

**Comment by CrazyFish444** [comment]
@Blightbuster I fixed it by going through all the FAQ questions and doing all the problem solving steps. It finally fixed when I did the waiting in the mechanic screen to open it for whatever reason so apparently the 2 errors are linked somehow... thanks for responing quickly and taking the time! 

TLDR: the error listen first can SOMETIMES be solved by fixing the ratscanner not scanning in the FAQ by sitting in mechanic and then opening.
